.github/workflows/iOS-pod-lint.yml

```diff
@@ -6,21 +6,69 @@ jobs:
     runs-on: macos-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Install Dependences
-      run: pod repo update
-      
-    - name: Lint FlipperKit
-      run: pod spec lint FlipperKit.podspec --use-libraries --allow-warnings --verbose --skip-import-validation
-    
+      - uses: actions/checkout@v2
```

Comment:
I remember that these are taking quite a long time and using up our resource pool. Do you think it would be safe to only validate these if the files are changed? We could add that as a spec to the "on" event at the top of the file.

Reply:
Thats a good idea. I will do that.

Reply:
These here are still not scoped to pod-related files. Is that intended?